app/queries/allocations_and_unalocated_users_query.rb

```diff
@@ -30,6 +30,6 @@ def where
   end
 
   def last_user_allocations
-    Allocation.group(:user_id).maximum(:end_at).values
+    Allocation.where(ongoing: true).group(:user_id).maximum(:end_at).values
```

Comment:
why not use the allocation scope, `.ongoing` ?

Reply:
I miss that one. Changing right now.